custom_visibility/src/custom/data/visibility/FilterOpportunities.php

```diff
@@ -168,23 +187,42 @@ protected function getTableAlias()
     }
 
     /**
-     * {@inheritdoc}
+     * implementation of ElasticStrategyInterface interface
+     *
+     * to create ealstic analyzer for the fields introduced here,
```

Comment:
I don't know what an elastic analyzer is.  Is that common knowledge?  Could we add a link to a reference or some words of explanation?